homeassistant/components/modern_forms/__init__.py

```diff
@@ -0,0 +1,198 @@
+"""The Modern Forms integration."""
+from __future__ import annotations
+
+import asyncio
+from datetime import timedelta
+import logging
+
+from aiomodernforms import (
+    ModernFormsConnectionError,
+    ModernFormsDevice,
+    ModernFormsError,
+)
+
+from homeassistant.components.fan import DOMAIN as FAN_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_MODEL, ATTR_NAME, ATTR_SW_VERSION, CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import ATTR_IDENTIFIERS, ATTR_MANUFACTURER, DOMAIN
+
+SCAN_INTERVAL = timedelta(seconds=5)
+PLATFORMS = [
+    FAN_DOMAIN,
+]
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Modern Forms component."""
+    return True
```

Comment:
I think this can just be omitted if not needed.

Reply:
removed